### PR TITLE
Multi distro travis

### PR DESCRIPTION
**Pull request by fmessmer**
replaces #139 

no kinetic_dev needed

### CODE REVIEW

**Comment by fmessmer** [comment]
requires https://github.com/ipa320/cob_robots/pull/626

**Comment by benmaidel** [comment]
Simulation works on xenial/kinetic. Just cob_extern needs to be `kinetic_dev` branch.

File: .travis.rosinstall.kinetic [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- git:
+    local-name: cob_gazebo_plugins
+    uri: 'https://github.com/ipa320/cob_gazebo_plugins.git'
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_hand.git'
+    local-name: cob-hand
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_manipulation.git'
```
Comment (NadiaHG) [inline .travis.rosinstall.kinetic:42]: why is cob_manipulation needed?

File: .travis.rosinstall.kinetic [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    local-name: cob_manipulation
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_navigation.git'
```
Comment (NadiaHG) [inline .travis.rosinstall.kinetic:46]: why is cob_navigation needed?

Comment (fmessmer) [reply .travis.rosinstall.kinetic:46]: `cob_simulation` requires `cob_robots` requires `cob_docking` requires `cob_navigation` (`cob_linear_nav`)

File: .travis.rosinstall.kinetic (hunk 1 above)
Comment (fmessmer) [reply .travis.rosinstall.kinetic:42]: `cob_simulation` requires `cob_robots` requires `cob_manipulation` (`cob_collision_monitor`)

Comment (NadiaHG) [reply .travis.rosinstall.kinetic:42]: we are not starting the collision_monitor with the cob_bringup , but you are right the dependency is listed

**Review by NadiaHG — APPROVED** [review]

**Review by NadiaHG — APPROVED** [review]

**Comment by fmessmer** [comment]
Travis for indigo succeeded
Travis for kinetic failed due to timeout...will merge anyway...as this will be fixed after first `cob_*` packages have been released for kinetic and thus, we can remove some entries from the .rosinstall.kinetic file

**Pull request merged**